pkg/logs/config/integration_config.go

```diff
@@ -50,8 +50,7 @@ type LogsProcessingRule struct {
 	ReplacePlaceholder string `mapstructure:"replace_placeholder"`
 	Pattern            string
 	// TODO: should be moved out
-	Reg                     *regexp.Regexp
-	ReplacePlaceholderBytes []byte
```

Comment:
It's nice to remove it but you are going to re-create the []byte each time you apply the processing rules, which is not efficient. I think **both** the regexp and the placeholder should be moved out of the configuration for that change to be interesting.